### ISSUE TITLE
Unsupported version of Gradle.

### ISSUE THREAD

**Issue by mintarasss** [GoranM/bdx#1043]
Hi,
When I try buld android apk file I get this error. But if I update gradle it still don’t work. Maybe you have a solution ?

> The project is using an unsupported version of Gradle.
> Please point to a supported Gradle version in the project's Gradle settings or in the project's Gradle wrapper (if applicable.)

p.s. 
Current version distributionUrl=http\://services.gradle.org/distributions/gradle-2.10-all.zip

**Comment by rafcolson** [comment]
I can't reproduce. What JDK are you using?

**Comment by mintarasss** [comment]
1.8.0

**Comment by rafcolson** [comment]
Well, that might be the problem then. If you want to deploy on Android, you need an earlier version (like [JDK7](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase7-521261.html)).

**Comment by rafcolson** [comment]
Java Development Kit 1.8.0?

**Comment by mintarasss** [comment]
Yes. i will try downgrade JDK 1.7. Tomorrow i will say if that helps ;)

**Comment by rafcolson** [comment]
If you don't have Oracle membership you can download JDK-7u79 for Windows x64 [here](https://www.dropbox.com/sh/oqvr4ihta71l6ai/AABhMhvO7j89IRV4jX8ue0Oda?dl=0).

**Comment by mintarasss** [comment]
Android studio does not allow me to set an older version of the JDK. Maybe you know how i can solve this?
![image](https://user-images.githubusercontent.com/26280733/38511737-ba4b6920-3c31-11e8-87d4-8a25da59cfd6.png)

**Comment by rafcolson** [comment]
If you installed your JDK, it should be enough to just edit the `JAVA_HOME` path in your system variables. Set the value to `C:\Program Files\Java\jdk1.7.0_79` or to whatever location is accurate on your pc. Try this, and it should work.

Newest versions of Android Studio probably require JDK 8 or higher. You could install an earlier version. But you don't actually need an Android Studio Bundle to deploy for Android with BDX, you just need SDK-Tools, which is shipped with the Bundle or [separately](https://developer.android.com/studio/index.html#command-tools).

**Comment by rafcolson** [comment]
@mintarasss 

Any success?
